lib/GameConstants.cpp

```diff
@@ -22,48 +22,48 @@
 #include <vcmi/HeroType.h>
 #include <vcmi/HeroTypeService.h>
 
-#include <vcmi/spells/Spell.h>
 #include <vcmi/spells/Service.h>
+#include <vcmi/spells/Spell.h>
 
-#include "VCMI_Lib.h"
-#include "mapObjects/CObjectClassesHandler.h"//todo: remove
-#include "CArtHandler.h"//todo: remove
-#include "CCreatureHandler.h"//todo: remove
-#include "spells/CSpellHandler.h" //todo: remove
-#include "CSkillHandler.h"//todo: remove
-#include "StringConstants.h"
-#include "CGeneralTextHandler.h"
-#include "CModHandler.h"//todo: remove
 #include "BattleFieldHandler.h"
+#include "CArtHandler.h"      //todo: remove
+#include "CCreatureHandler.h" //todo: remove
+#include "CGeneralTextHandler.h"
+#include "CModHandler.h"   //todo: remove
+#include "CSkillHandler.h" //todo: remove
 #include "ObstacleHandler.h"
+#include "StringConstants.h"
+#include "VCMI_Lib.h"
+#include "mapObjects/CObjectClassesHandler.h" //todo: remove
+#include "spells/CSpellHandler.h"             //todo: remove
 
 VCMI_LIB_NAMESPACE_BEGIN
 
 const SlotID SlotID::COMMANDER_SLOT_PLACEHOLDER = SlotID(-2);
-const SlotID SlotID::SUMMONED_SLOT_PLACEHOLDER = SlotID(-3);
-const SlotID SlotID::WAR_MACHINES_SLOT = SlotID(-4);
-const SlotID SlotID::ARROW_TOWERS_SLOT = SlotID(-5);
+const SlotID SlotID::SUMMONED_SLOT_PLACEHOLDER  = SlotID(-3);
+const SlotID SlotID::WAR_MACHINES_SLOT          = SlotID(-4);
+const SlotID SlotID::ARROW_TOWERS_SLOT          = SlotID(-5);
 
-const PlayerColor PlayerColor::SPECTATOR = PlayerColor(252);
+const PlayerColor PlayerColor::SPECTATOR        = PlayerColor(252);
 const PlayerColor PlayerColor::CANNOT_DETERMINE = PlayerColor(253);
-const PlayerColor PlayerColor::UNFLAGGABLE = PlayerColor(254);
-const PlayerColor PlayerColor::NEUTRAL = PlayerColor(255);
-const PlayerColor PlayerColor::PLAYER_LIMIT = PlayerColor(PLAYER_LIMIT_I);
-const TeamID TeamID::NO_TEAM = TeamID(255);
+const PlayerColor PlayerColor::UNFLAGGABLE      = PlayerColor(254);
+const PlayerColor PlayerColor::NEUTRAL          = PlayerColor(255);
+const PlayerColor PlayerColor::PLAYER_LIMIT     = PlayerColor(PLAYER_LIMIT_I);
+const TeamID      TeamID::NO_TEAM               = TeamID(255);
 
 namespace GameConstants
 {
 #ifdef VCMI_NO_EXTRA_VERSION
 	const std::string VCMI_VERSION = "VCMI " VCMI_VERSION_STRING;
 #else
-	const std::string VCMI_VERSION = "VCMI " VCMI_VERSION_STRING "." + std::string{GIT_SHA1};
+	const std::string VCMI_VERSION = "VCMI " VCMI_VERSION_STRING "." + std::string{ GIT_SHA1 };

```

Comment:
personally I don't like wrapping with spaces, but not insisting on either way

Reply:
I don't mind about this particular case, but it looks like same option also applies to all braced lists. From docs:
Cpp11BracedListStyle 
```
true:                                  false:
vector<int> x{1, 2, 3, 4};     vs.     vector<int> x{ 1, 2, 3, 4 };
vector<T> x{{}, {}, {}, {}};           vector<T> x{ {}, {}, {}, {} };
f(MyMap[{composite, key}]);            f(MyMap[{ composite, key }]);
new int[3]{1, 2, 3};                   new int[3]{ 1, 2, 3 };
```
Personally, for actual lists with multiple items I'd rather keep spaces

Reply:
yes, for multiple items that's fine

Reply:
For me left part looks better

Reply:
I would better use vector x = {1, 2, 3, 4}